notify/alerts.go

```diff
@@ -220,7 +221,7 @@ func parseReceivers(receivers string) (*regexp.Regexp, error) {
 func parseFilter(filter []string) ([]*labels.Matcher, error) {
 	matchers := make([]*labels.Matcher, 0, len(filter))
 	for _, matcherString := range filter {
-		matcher, err := labels.ParseMatcher(matcherString)
+		matcher, err := compat.Matcher(matcherString, "unknown")
```

Comment:
This is called from `GetAlerts` and `GetAlertGroups` in `grafana/grafana` which in most cases is for API calls. I didn't see calls to these functions outside of the Alertmanager API or tests.